src/components/Editor/Resources/ResourceListSearch.vue

```diff
@@ -84,6 +114,22 @@ export default {
 		noResult() {
 			return this.$t('calendar', 'No match found')
 		},
+		hasAdvancedFilters() {
+			return semver.gte(semver.coerce(OC.config.version), '23.0.0')
```

Comment:
Probably overkill to bring semver just for that. Split on `.`, take first element, parse as int and compare. 

Also leave a TODO comment like 
```
// Remove me when Calendar doesn't support server < 23
```

Reply:
I dropped semver and added a comment.